packages/neos-ui/src/Containers/SecondaryToolbar/DimensionSwitcher/index.js

```diff
@@ -79,13 +79,37 @@ export default class DimensionSwitcher extends PureComponent {
         activePresets: new Map()
     };
 
+    state = {
+        isOpen: false
+    };
+
+    handleSelectPreset = preset => {
+        this.setState({isOpen: false});
+        this.props.selectPreset(preset);
```

Comment:
Looking at https://github.com/neos/neos-ui/blob/master/packages/neos-ui-redux-store/src/CR/ContentDimensions/index.js#L21, this needs to take a couple of more arguments :)

Reply:
Ouch! Thanks!